src/popover/popover.jsx

```diff
@@ -0,0 +1,353 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WindowListenable from '../mixins/window-listenable';
+import RenderToLayer from '../render-to-layer';
+import StylePropable from '../mixins/style-propable';
+import Extend from '../utils/extend';
+import CssEvent from '../utils/css-event';
+import Dom from '../utils/dom';
+import PropTypes from '../utils/prop-types';
+import Transitions from '../styles/transitions';
+import Paper from '../paper';
+import throttle from 'lodash.throttle';
+import AutoPrefix from '../styles/auto-prefix';
+import ContextPure from '../mixins/context-pure';
+
+export default React.createClass({
```

Comment:
is https://github.com/chrismcv/material-ui/commit/2414ad47e1c84d6044f8b23995c09de11c93b47a what you mean?